.ansible-lint

```diff
@@ -1,3 +1,6 @@
+skip_list:
+- '503'
+- '403'
```

Comment:
I think it's better to use `warn_list`.

Reply:
Somehow warn_list was not working correctly for me, therefore added skip_list